src/error.js

```diff
@@ -376,13 +376,14 @@ export function maybeReportErrorToViewer(win, data) {
  * @visibleForTesting
  */
 export function errorReportingDataForViewer(errorReportData) {
+  const {m, a, s, el, v, jse} = errorReportData;
   return dict({
-    'm': errorReportData['m'], // message
-    'a': errorReportData['a'], // isUserError
-    's': errorReportData['s'], // error stack
-    'el': errorReportData['el'], // tagName
-    'v': errorReportData['v'], // runtime
-    'jse': errorReportData['jse'], // detectedJsEngine
+    m, // message
+    a, // isUserError
+    s, // error stack
+    el, // tagName
+    v, // runtime
+    jse, // detectedJsEngine
```

Comment:
Does this avoid the binary size increase in the original PR?

Reply:
still need to make the comparison. Will update later. Thanks for the input.

Reply:
w/o change: PASS  ./dist/v0.js: 81.21KB < maxSize 81.26KB (gzip)
with change: PASS  ./dist/v0.js: 81.22KB < maxSize 81.26KB (gzip)

ran 
>> gulp dist --fortesting --noextensions && gulp bundle-size

So there are no savings. This reads better, so let's keep.

Reply:
I think you'll still need string keys without an extern. Let's just revert this and keep the test changes. 

You still need to recreate this PR to merge to the correct amphtml branch.

Reply:
acknowledged. Thanks